package.json

```diff
@@ -29,87 +29,87 @@
         "webpack-merge": "5.8.0"
     },
-    "dependencies": {
-        "@azure/msal-browser": "^2.24.0",
-        "@braintree/sanitize-url": "6.0.0",
-        "@monaco-editor/loader": "^1.3.2",
-        "@paperbits/azure": "0.1.515",
-        "@paperbits/common": "0.1.515",
-        "@paperbits/core": "0.1.515",
-        "@paperbits/prosemirror": "0.1.515",
-        "@paperbits/styles": "0.1.515",
-        "@webcomponents/custom-elements": "1.5.0",
-        "@webcomponents/shadydom": "^1.9.0",
-        "applicationinsights-js": "^1.0.21",
-        "client-oauth2": "4.3.3",
-        "codemirror": "^5.65.5",
-        "core-js": "^3.22.8",
-        "d3": "^7.4.4",
-        "google-maps": "^4.3.3",
-        "graphql": "^15.5.0",
-        "graphql-config": "^4.0.1",
-        "graphql-language-service": "^3.1.4",
-        "idb-keyval": "^6.1.0",
-        "js-beautify": "^1.14.3",
-        "knockout": "^3.5.1",
-        "knockout-mapping": "^2.6.0",
-        "knockout.validation": "^2.0.4",
-        "liquidjs": "^9.37.0",
-        "lunr": "^2.3.9",
-        "mime": "^3.0.0",
-        "moment": "^2.29.3",
-        "monaco-editor": "^0.29.1",
-        "msal": "^1.4.16",
-        "prettier": "^2.6.2",
-        "prismjs": "^1.28.0",
-        "rehype-raw": "^6.1.1",
-        "rehype-sanitize": "^5.0.1",
-        "rehype-stringify": "^9.0.3",
-        "remark": "^14.0.2",
-        "remark-gfm": "^3.0.1",
-        "remark-parse": "^10.0.1",
-        "remark-rehype": "^10.1.0",
-        "saxen": "^8.1.2",
-        "topojson-client": "^3.1.0",
-        "truncate-html": "^1.0.4"
-    }
+  "dependencies": {
+    "@azure/msal-browser": "^2.26.0",
+    "@braintree/sanitize-url": "6.0.0",
+    "@monaco-editor/loader": "^1.3.2",
+    "@paperbits/azure": "0.1.516",
+    "@paperbits/common": "0.1.516",
+    "@paperbits/core": "0.1.516",
+    "@paperbits/prosemirror": "0.1.516",
+    "@paperbits/styles": "0.1.516",
+    "@webcomponents/custom-elements": "1.5.0",
+    "@webcomponents/shadydom": "^1.9.0",
+    "applicationinsights-js": "^1.0.21",
+    "client-oauth2": "4.3.3",
+    "codemirror": "^5.65.5",
+    "core-js": "^3.23.2",
+    "d3": "^7.4.4",
+    "google-maps": "^4.3.3",
+    "graphql": "^15.5.0",
+    "graphql-config": "^4.0.1",
+    "graphql-language-service": "^3.1.4",
+    "idb-keyval": "^6.2.0",
+    "js-beautify": "^1.14.4",
+    "knockout": "^3.5.1",
+    "knockout-mapping": "^2.6.0",
+    "knockout.validation": "^2.0.4",
+    "liquidjs": "^9.37.0",
+    "lunr": "^2.3.9",
+    "mime": "^3.0.0",
+    "moment": "^2.29.3",
+    "monaco-editor": "^0.29.1",
+    "msal": "^1.4.16",
+    "prettier": "^2.7.1",
+    "prismjs": "^1.28.0",
+    "rehype-raw": "^6.1.1",
+    "rehype-sanitize": "^5.0.1",
+    "rehype-stringify": "^9.0.3",
+    "remark": "^14.0.2",
+    "remark-gfm": "^3.0.1",
+    "remark-parse": "^10.0.1",
+    "remark-rehype": "^10.1.0",
+    "saxen": "^8.1.2",
+    "topojson-client": "^3.1.0",
+    "truncate-html": "^1.0.4"
+  }
```

Comment:
Could you please take a look how/why is it changing the indentation of the "dependencies" to two spaces while all other keys are indented with four? Or are you doing this on purpose for some reason?
It's not huge issue, it's just producing unnecessary changes every time and I believe it should be indented the same way in whole file.

Reply:
Applied formatting.